src/client/South/ConfigurePoints.jsx

```diff
@@ -171,7 +171,7 @@ const ConfigurePoints = () => {
     const name = `points.${index}.${key}`
     return (
       /* eslint-disable-next-line react/jsx-props-no-spreading */
-      { name, value: <Control onChange={onChange} name={name} {...rest} /> }
+      { name, value: <Control onChange={onChange} name={name} point {...rest} /> }
```

Comment:
I'm not sure to understand the 'point' props : in OIbScanMode component, it is marked as a boolean but here it seems to me it is an object.